bower_components/yeast

```diff
@@ -0,0 +1 @@
+/Users/andre/.local/share/bower/links/yeast
```

Comment:
Do we want dev specific references in the codebase?

Reply:
Fixed!